src/actions/actions.ts

```diff
@@ -2694,3 +2695,65 @@ class ToggleCaseAndMoveForward extends BaseMovement {
     return position.getRight();
   }
 }
+
+abstract class IncrementDecrementNumberAction extends BaseMovement {
+  modes = [ModeName.Normal];
+  canBePrefixedWithCount = true;
+
+  offset: number;
+
+  public async execActionWithCount(position: Position, vimState: VimState, count: number): Promise<Position> {
+    count = count || 1;
+    const text = TextEditor.getLineAt(position).text;
+
+    // They may start on a number, so check the current word boundaries first.
+
+    for (let { start, end, word } of Position.IterateWords(position.getWordLeft(true))) {
+      // '-' doesn't count as a word, but is important to include in parsing the number
+      if (text[start.character - 1] === '-') {
```

Comment:
Oh geez, that actually seems like it could have some bad interactions with people who DONT use - as a word delimiter. It's also really annoying. Oh well, it's fine for now.